app/models/project.rb

```diff
@@ -40,6 +40,8 @@ class Project < ActiveRecord::Base
   has_many :tagged_resources, as: :resource
   has_many :tags, through: :tagged_resources
 
+  enum state: [:paused, :finished]
```

Comment:
you should be able to proxy the finished method via a super call and default back to the current implementation by the way.